src/FixedDataTableCellGroup.js

```diff
@@ -143,11 +228,12 @@ class FixedDataTableCellGroupImpl extends React.Component {
 
     return (
       <FixedDataTableCell
-        isScrolling={this.props.isScrolling}
+        isScrolling={props.isScrolling}
         align={columnProps.align}
         className={className}
-        height={height}
-        key={key}
+        height={props.rowHeight}
+        columnIndex={columnIndex}
+        key={staticIndex}
```

Comment:
*this is where column virtualization kicks in*

since staticIndex reflects the IntegerBufferSet's buffer mapping (#461) for v-cells, there will be minimal key changes, thus letting React efficiently reuse the same component for different columns